### PR TITLE
feat(timepicker): add styles

### PR DESCRIPTION
**Pull request by skhamvon**

### CODE REVIEW

**Review by manoncarbonnel — APPROVED** [review]

File: packages/components/timepicker/src/components/osds-timepicker/osds-timepicker.tsx [hunk 1]
```diff
@@ -86,10 +88,12 @@ export class OsdsTimepicker implements OdsTimepickerAttribute {
                     type={ ODS_INPUT_TYPE.time }
                     value={ value }
                     ref={(el: HTMLElement) => this.input = el as HTMLInputElement}
+                    color={ "primary" }
         >
         </osds-input>
         {
           (currentTimezone || timezones) && <osds-select
+            class={ "osds-select" }
```
Comment (aesteves60) [inline packages/components/timepicker/src/components/osds-timepicker/osds-timepicker.tsx:96]: binding is not necessary

File: packages/components/timepicker/src/components/osds-timepicker/osds-timepicker.tsx [hunk 2]
```diff
@@ -86,10 +88,12 @@ export class OsdsTimepicker implements OdsTimepickerAttribute {
                     type={ ODS_INPUT_TYPE.time }
                     value={ value }
                     ref={(el: HTMLElement) => this.input = el as HTMLInputElement}
+                    color={ "primary" }
```
Comment (aesteves60) [inline packages/components/timepicker/src/components/osds-timepicker/osds-timepicker.tsx:91]: binding is not necessary

**Review by aesteves60 — CHANGES_REQUESTED** [review]

File: packages/components/timepicker/src/components/osds-timepicker/osds-timepicker.tsx (hunk 1 above)
Comment (skhamvon) [reply packages/components/timepicker/src/components/osds-timepicker/osds-timepicker.tsx]: changed

File: packages/components/timepicker/src/components/osds-timepicker/osds-timepicker.tsx (hunk 2 above)
Comment (skhamvon) [reply packages/components/timepicker/src/components/osds-timepicker/osds-timepicker.tsx]: changed

File: packages/components/timepicker/src/components/osds-timepicker/osds-timepicker.scss [hunk 3]
```diff
@@ -1,3 +1,15 @@
-:host {
-  font-weight: normal;
+.osds-timepicker {
+  :host(&#{&}--inline) {
+    display: inline-flex;
+
+    & .osds-select {
```
Comment (dpellier) [inline packages/components/timepicker/src/components/osds-timepicker/osds-timepicker.scss:5]: we go with BEM on ODS, this should be more like:
```
:host(.osds-timepicker) {
  ...host only code
}

.osds-timepicker {
  &__timezones {
    ... select code
  }
}
```

Comment (skhamvon) [reply packages/components/timepicker/src/components/osds-timepicker/osds-timepicker.scss]: Class has been changed

**Review by dpellier — CHANGES_REQUESTED** [review]
please also add the `lint:ts` command and fix the issues

**Review by dpellier — CHANGES_REQUESTED** [review]
Issue on flex, timezone select should be aligned
Capture d’écran 2023-11-07 à 09 17 57

**Review by dpellier — CHANGES_REQUESTED** [review]
Did you check if there is an array command instead for this control?
Capture d’écran 2023-11-07 à 09 18 58
Capture d’écran 2023-11-07 à 09 19 06

**Review by astagnol — COMMENTED** [review]
As said previously, on flex input and timezone select should be aligned and it should go 50/50 (checked with Design team)

**Review by dpellier — CHANGES_REQUESTED** [review]
If I set a timezone and I click on it in the select, it does not get selected
Capture d’écran 2023-11-08 à 11 02 39

**Review by dpellier — CHANGES_REQUESTED** [review]
Timezone trigger should be hidden if the attribute is set as an empty array

File: packages/components/timepicker/src/components/osds-timepicker/osds-timepicker.spec.ts [hunk 4]
```diff
@@ -42,143 +42,143 @@ describe('spec:osds-timepicker', () => {
...
 
   describe('attributes', () => {
     const config = {
-      instance: () => instance,
-      page: () => page,
-      root: () => page.root,
-      wait: () => page.waitForChanges(),
+      instance: ():OsdsTimepicker => instance,
+      page: ():SpecPage => page,
+      root: ():HTMLElement |undefined => page.root,
```
Comment (dpellier) [inline packages/components/timepicker/src/components/osds-timepicker/osds-timepicker.spec.ts:93]: you can ref SpecPage['root'] directly

**Review by dpellier — CHANGES_REQUESTED** [review]

**Review by dpellier — APPROVED** [review]

**Review by aesteves60 — APPROVED** [review]

**Review by astagnol — APPROVED** [review]

**Pull request merged**